src/intl/en.json

```diff
@@ -155,5 +155,60 @@
   "language-swedish": "Swedish",
   "language-turkish": "Turkish",
   "language-chinese": "Chinese Simplified",
-  "website-last-updated": "Website last updated"
+  "website-last-updated": "Website last updated",
+  "whats-ethereum-meta-title": "What is Ethereum?",
```

Comment:
Can we continue with the `page-` pattern? e.g. so start all these with `page-what-is-ethereum-...`